detail.md

```diff
@@ -0,0 +1,3761 @@
+## Detail
+
+> **Note**: This section is still under construction.
+> For the purposes of implementation, if any details below differ from the described request flows above, the section below SHOULD be corrected.
+> When they match, this note SHOULD be removed.
+
+The behavior of the endpoints are covered in detail in this section, organized by route and entity.
```

Comment:
route and entity? 

We should define route and entity before using them here as the main introduction for the section I think.
`route: API protocol flow consisting of requests and responses between two entities implementing the API. `

Reply:
Done